include/circt/Dialect/MSFT/MSFTAttributes.td

```diff
@@ -61,3 +61,30 @@ def PhysLocation : MSFT_Attr<"PhysLocation"> {
     "PrimitiveTypeAttr":$primitiveType,
     "uint64_t":$x, "uint64_t":$y, "uint64_t":$num);
 }
+
+def PhysicalBounds : MSFT_Attr<"PhysicalBounds"> {
+  let summary = "Describes a rectangle bounding a physical region on a device";
+  let mnemonic = "physical_bounds";
+  let parameters = (ins
```

Comment:
What should we do for "unbounded" boxes? Is the thing which creates this expected to know the maximum y,x and use that? That's a totally reasonable approach, IMO. If so, that should be discussed in the description.

Reply:
Yes my intention is whatever creates these has to specify lower and upper bounds in the x and y axes. I can document this.